packages/react-reconciler/src/__tests__/ReactSuspense-test.internal.js

```diff
@@ -1020,35 +1030,35 @@ describe('ReactSuspense', () => {
         );
       }
 
-      const root = ReactTestRenderer.create(<App />);
+      ReactDOM.render(<App />, container);
       assertLog(['Suspend! [Tab: 0]', ' + sibling', 'Loading...']);
-      expect(root).toMatchRenderedOutput('Loading...');
+      expect(container.textContent).toEqual('Loading...');
 
       await resolveText('Tab: 0');
-      await waitForPaint(['Tab: 0']);
-      expect(root).toMatchRenderedOutput('Tab: 0 + sibling');
+      assertLog(['Tab: 0']);
```

Comment:
Any concerns with the `waitForPaint` calls moving to sync `assertLog` for legacy render tests? The text resolution already completes and populates scheduler logger when using ReactDOM.render as oppose to ReactTestRenderer.create() but I'm not sure of the reason.